railties/test/application/configuration_test.rb

```diff
@@ -97,7 +97,9 @@ def restore_default_config
     end
 
     def suppress_sqlite3_warning
-      add_to_config "config.active_record.sqlite3_production_warning = false"
+      add_to_config <<-RUBY
```

Comment:
This change is explained here, but essentially not using the heredoc form won't preserve the indentation which turns into:

```ruby
puts File.read("#{app_path}/config/application.rb")
# ...
module AppTemplate
  class Application < Rails::Application
    config.load_defaults 7.2
    # ...
        config.eager_load = false
        config.active_record.sqlite3_production_warning = false
        config.active_support.cache_format_version = 7.1
config.active_record.sqlite3_production_warning = false
config.logger = MyLogRecorder.new
# ...
```

Not a huge deal since no one really looks at the generated config for test apps, but it'd be nice to keep them consistent.